docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
+help in exchange for the food that your members will purchase.
 
-If you're having difficulty finding a location, ask on the `Slack <http://slack.writethedocs.org>`_: you have an international community ready to help.
 
 Find attendees and speakers
 ---------------------------
 
 Attendees:
 ~~~~~~~~~~
 
-Once your meetup page is up, start promoting it. To do that:
-
-* Announce your meetup on `Write the Docs Slack <https://writethedocs.slack.com>`_ and join meetup-organizers and meetup-general channels. You can also create a dedicated channel to your meetup.
-* Announce your meetup on `Write the Docs Forum <http://forum.writethedocs.org/c/meetups>`_.
-* Create a Twitter account and connect it to your meetup's page (`example <https://twitter.com/WriteTheDocsPDX/>`_). Use it to announce your next events, live tweet your meetup, post pictures, reach new people, retweet other Write the Docs meetups, etc.
-* Use your network: ask your colleagues, friends, and contacts to promote your meetup.
-* Reach new people: find other community leaders and ask them to spread the word.
-* Attend other meetups in your area and network.
-* Add your meetup to local tech events calendars.
+Once you've set up your Meetup, promote it!
 
-No-show:
-^^^^^^^^
+* Use your network: ask colleagues, friends, and contacts to promote the meetup.
+* Attend other meetups. Tell them about Write The Docs.
+* Add your meetup to local tech event calendars.
+* Use social media. Several Write The Docs meetups publicize through Twitter (`example <https://twitter.com/WriteTheDocsPDX/>`_).
+* Help your fellow Meetup leaders. Retweet their announcements.
+* Announce your meetup on `Write the Docs Slack <https://writethedocs.slack.com>`_.
 
-A lot of people RSVPed but didn't show? Don't worry, that's one of the downsides of free meetups. Do a head-count at each event to know how many people usually attend your meetup.
 
-If something went wrong, listen to feedback and make changes. Try to have a different topic and speakers for each meetup: if you don't do that, people will get bored and won't come anymore.
 
 Speakers:
 ~~~~~~~~~
 
-To find speakers:
-
-* Look at previous `Write the Docs conferences <http://www.writethedocs.org/conf/>`_ and other `Write the Docs meetups <http://www.writethedocs.org/meetups/>`_.
-* Look at other tech meetups and conferences in your area. Don't limit yourself to writers.
-* Ask around on `Write the Docs Slack <https://writethedocs.slack.com>`_. Don't forget to check local channels.
-* Invite your meetup's members to do a talk: you can do a CFP by email or ask people during your event.
-* Ask on Twitter.
-
-What to Do During a Meetup?
----------------------------
 
-To help you get a better idea of what you could do, links in this section lead to different Write the Docs meetups that already took place.
+* Look for previous and future speakers at `Write the Docs conferences <http://www.writethedocs.org/conf/>`_ .
```

Comment:
This could be a `:doc:`/conf/index` link (or we could create a reference for even more lasting semantic value).